crud.py

```diff
@@ -0,0 +1,68 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(db: Session, skip: int, limit: int):
+    return (
+        db.query(models.DBAuthor).offset
+        (offset=skip).limit(limit=limit).all()
```

Comment:
The `offset` method should be called with parentheses, i.e., `offset(skip)`, instead of `offset=skip`. This is a syntax error and will cause the code to fail.